core/new-gui/src/app/workspace/component/property-editor/breakpoint-property-edit-frame/breakpoint-property-edit-frame.component.ts

```diff
@@ -35,15 +29,13 @@ import { filter } from "rxjs/operators";
  * We use library `@ngx-formly` to generate form from json schema
  * https://github.com/ngx-formly/ngx-formly
  */
+@UntilDestroy()
 @Component({
   selector: "texera-breakpoint-frame",
   templateUrl: "./breakpoint-property-edit-frame.component.html",
   styleUrls: ["./breakpoint-property-edit-frame.component.scss"]
 })
-export class BreakpointPropertyEditFrameComponent
-  implements OnInit, OnDestroy, OnChanges {
-  subscriptions = new Subscription();
-
+export class BreakpointPropertyEditFrameComponent implements OnInit {
```

Comment:
can you fix the github warning below? it says this class needs to implement OnChanges

Reply:
fixed. this is due to merge error, omitting implementation of `OnChanges`.